docs/contributing/style_guides/code_style.md

```diff
@@ -67,18 +67,10 @@ pyupgrade <PATH/TO/YOUR/CHANGES> --py3-plus
 
 ### Type Checking
 
-Our CI system will perform static type-checking using [mypy](https://mypy.readthedocs.io/en/stable/index.html#) on select modules & packages.
+Our CI system will perform static type-checking using [mypy](https://mypy.readthedocs.io/en/stable/index.html#).
 
-Type-checked modules.
-
-```
-data_asset
-exceptions
-jupyter_ux
-profile
-self_check
-types
-```
+`contrib` and other select `great_expectations/` packages are excluded from type-checking.
+See the `mypy` section in [`pyproject.toml`](https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L15) for more details.
```

Comment:
I would remove `#L15` from the link since this file may change and that part may fall out of date.